solidity/test/BondedECDSAKeepTest.js

```diff
@@ -1658,16 +1666,18 @@ contract("BondedECDSAKeep", (accounts) => {
       await tokenStaking.setBeneficiary(member1, beneficiary)
       await tokenStaking.setBeneficiary(member2, beneficiary)
 
-      await keep.distributeERC20Reward(token.address, valueWithRemainder)
+      await keep.distributeERC20Reward(token.address, valueWithRemainder, {
+        from: accounts[0],
+      })
 
       // Check balances of all keep members' and beneficiary.
       const newBalances = await getERC20BalancesFromList(accountsInTest, token)
       assert.equal(newBalances.toString(), expectedBalances.toString())
     })
 
     async function initializeTokens(token, keep, account, amount) {
-      await token.mint(account, amount)
-      await token.approve(keep.address, amount)
+      await token.mint(account, amount, {from: accounts[0]})
+      await token.approve(keep.address, amount, {from: accounts[0]})
```

Comment:
I think we should be doing `await token.approve(keep.address, amount, {from: account})` here.

We are minting tokens to `account` and that `account` should approve tokens.

Reply:
Yup, you're right. Changed.